CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -236,6 +236,18 @@ local function CreateSettingsHub()
 			ZIndex = SETTINGS_BASE_ZINDEX,
 			Parent = this.ClippingShield
 		};
+		this.VRShield = utility:Create("Frame") {
+			Name = "VRBackground",
+			Parent = this.Shield,
+
+			BackgroundColor3 = Color3.new(0.2, 0.2, 0.2),
```

Comment:
Are we able to use SETTINGS_SHIELD_COLOR for this shield as well? As well as SETTINGS_SHIELD_TRANSPARENCY or are they different values?
If they are different we could create some const for them at the top too.